source/_posts/2015-07-17-operational-tcpdump-handler-now-open-source.markdown

```diff
@@ -0,0 +1,24 @@
+---
+layout: post
+title: "Operational tcpdump Handler Now Open Source"
+date: 2015-07-17
+tags: [engineering]
+---
+
+Written by [Aaron Blythe](engineers/aaron-blythe) and [David Crowder](engineers/david-crowder)
```

Comment:
@david-crowder needs a bio if you're adding links. Also should update the index.html for the index of Engineers